src/main/java/com/thealgorithms/datastructures/graphs/Kosaraju.java

```diff
@@ -116,7 +145,6 @@ private void dfs2(int node, int vis[], List<List<Integer>> list){
         scc.add(node);
     }
 
-
     public static void main(String[] args) {
```

Comment:
Please remove main, it's redundant because there are tests checking that the algorithm works